src/Components/Directory.Build.targets

```diff
@@ -78,7 +78,7 @@
   <Target Name="CalculateConfigurationSchemaGeneratorPath">
     <MSBuild Projects="$(ConfigurationSchemaGeneratorProjectPath)"
              Targets="GetTargetPath"
-             RemoveProperties="TargetFramework">
+             Properties="$(Properties);TargetFramework=$(DefaultTargetFramework)">
```

Comment:
I'm not sure this is right. (actually I'm pretty sure it is wrong). Can you explain why you are proposing this change? What doesn't work about the current code?

Reply:
Reverted with the latest changes.